python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
-# Create the Monte-Carlo sample.
+.. math::
 
-# %%
-sampleSize = 100
-inputSample = inputDistribution.getSample(sampleSize)
-outputStress = g(inputSample)
-outputStress[0:5]
+   \\theta = (R,C,\\gamma).
 
-# %%
-# Plot the histogram of the output.
+We consider a data set where the number of observations is equal to:
 
-# %%
-histoGraph = ot.HistogramFactory().build(outputStress / 1.0e6).drawPDF()
-histoGraph.setTitle("Histogram of the sample stress")
-histoGraph.setXTitle("Stress (MPa)")
-histoGraph.setLegends([""])
-view = viewer.View(histoGraph)
+.. math::
 
-# %%
-# Generate observation noise.
+   n = 10.
 
-# %%
-stressObservationNoiseSigma = 10.0e6  # (Pa)
-noiseSigma = ot.Normal(0.0, stressObservationNoiseSigma)
-sampleNoiseH = noiseSigma.getSample(sampleSize)
-observedStress = outputStress + sampleNoiseH
 
-# %%
-observedStrain = inputSample[:, 0]
+The observations are the pairs :math:`\\{(\\epsilon_i,\\sigma_i)\\}_{i=1,...,n}`,
+i.e. each observation is a couple made of the strain and the corresponding stress.
+
+In the particular situation where we want to calibrate this model, the following
+list presents which variables are observed input variables, input calibrated
+variables and observed output variables.
+
+- :math:`\\epsilon`: Input. Observed.
+- :math:`R`, :math:`C`, :math:`\\gamma` : Inputs. Calibrated.
+- :math:`\\sigma`: Output. Observed.
+"""
 
 # %%
-graph = ot.Graph("Observations", "Strain", "Stress (MPa)", True)
-cloud = ot.Cloud(observedStrain, observedStress / 1.0e6)
-graph.add(cloud)
-view = viewer.View(graph)
+import openturns as ot
+import openturns.viewer as viewer
+from matplotlib import pylab as plt
+from openturns.usecases import chaboche_model
+
+ot.Log.Show(ot.Log.NONE)
 
 # %%
-# Set the calibration parameters
-# ------------------------------
+# Define the observations
+# -----------------------
+# In practice, we generally use a data set which has been obtained from
+# measurements.
+# This data set can be loaded using e.g. :meth:`~openturns.Sample.ImportFromCSVFile`.
+# Here we define the data directly.
+cm = chaboche_model.ChabocheModel()
+print(cm.data)
+observedStrain = cm.data[:, 0]
+observedStress = cm.data[:, 1]
+nbobs = cm.data.getSize()
 
 # %%
-# Define the value of the reference values of the :math:`\theta` parameter. In the bayesian framework, this is called the mean of the *prior* gaussian distribution. In the data assimilation framework, this is called the *background*.
+# Print the Chaboche model
+print("Inputs:", cm.model.getInputDescription())
+print("Outputs:", cm.model.getOutputDescription())
 
 # %%
+# We see that there are four inputs: `Strain`, `R`, `C` and `Gamma` and one
+# output: `Sigma`.
+# The `Strain` is observed on input and the stress `Sigma` is observed on
+# output.
+# Using these observations, we want to estimate the parameters `R`, `C`
+# and `Gamma`.
+# %%
+# Set the calibration parameters
+# ------------------------------
+# In this part, we begin the calibration study.
```

Comment:
```suggestion
#
# Set the calibration parameters
# ------------------------------
#
# In this part, we begin the calibration study.
```
This part is not displayed correctly

Reply:
Ok. Fixed. Also, I tried the "r" trick on the docstrings in the header to avoid the double backslashes, as suggested by @jschueller. This works and is efficient. Thanks to @jschueller for the trick!